js/poi.js

```diff
@@ -164,6 +164,8 @@ function POI(tags) {
             return '🍸';
         case 'pub':
             return '🍺';
+        case 'ice_cream':
+            return '🍨';
```

Comment:
Can you please move this to a separate PR (in order to make it easier to review)?

Reply:
Done.

Reply:
Thanks, feel free to submit another PR for it!